onnxscript/test/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -226,6 +235,16 @@ def wrapped(fn):
 
 
 SKIP_SUBTESTS: tuple[DecorateMeta, ...] = (
+    skip(
+        "div",
+        matcher=lambda sample: sample.kwargs.get("rounding_mode") is not None,
+        reason="rounding_mode=True is not supported",
```

Comment:
rounding_mode seems to be a string. How about

```suggestion
        reason="rounding_mode is not yet supported",
```

We can add support for it in a future PR.